src/lib/components/Swap/Status/StatusDialog.tsx

```diff
@@ -68,12 +70,18 @@ interface TransactionStatusProps {
 }
 
 function TransactionStatus({ tx, onClose }: TransactionStatusProps) {
+  const { chainId } = useActiveWeb3React()
+
   const Icon = useMemo(() => {
     return tx.receipt?.status ? CheckCircle : Spinner
   }, [tx.receipt?.status])
   const heading = useMemo(() => {
     return tx.receipt?.status ? <Trans>Transaction submitted</Trans> : <Trans>Transaction pending</Trans>
   }, [tx.receipt?.status])
+  const etherscanLink = useMemo(
+    () => getExplorerLink(chainId || 1, tx.info.response.hash, ExplorerDataType.TRANSACTION),
```

Comment:
1 should be SupportedChainId.MAINNET. Even better would be if it were defaulted by getExplorerLink.